### DISCUSSION TITLE
Insufficient Quota when using OpenAI's API Key

### DISCUSSION THREAD

**Discussion by vineeth-balusani** [mem0ai/mem0#1236] [Q&A]
I don't have the premium subscription of ChatGPT, but I got the API Key from my account and trying to use it, however, I am getting the following error 
![image](https://github.com/embedchain/embedchain/assets/15002537/d2b81454-d214-44a8-af24-ac74e83295a6)
Do I need to have subscription to GPT4 for it to work? I am able to use GPT3.5 when tried directly in https://chat.openai.com/

**Comment by smtejas7** [comment]
facing same issue, did u find the solution for this above ?
